root/usr/bin/discovery-menu

```diff
@@ -0,0 +1,576 @@
+#!/bin/ruby
+require "newt"
+require "discovery"
+require "facter"
+require "ipaddr"
+
+# disable GC until https://github.com/theforeman/ruby-newt/issues/2 is fixed
+GC.disable
+
+def fdi_version file = 'VERSION'
+  IO.read("/usr/share/fdi/#{file}").chomp rescue 'GIT'
```

Comment:
rescue should not be encouraged, please check if the file exists.
rescue on the entire method as last option.